L/LLVM/common.jl

```diff
@@ -469,11 +469,7 @@ function configure_build(ARGS, version; experimental_platforms=false, assert=fal
     end
     if version >= v"12"
         push!(products, LibraryProduct("libclang-cpp", :libclang_cpp, dont_dlopen=true))
-        push!(products, ExecutableProduct("lld", :lld, "tools"))
-        push!(products, ExecutableProduct("ld.lld", :ld_lld, "tools"))
-        push!(products, ExecutableProduct("ld64.lld", :ld64_lld, "tools"))
-        push!(products, ExecutableProduct("lld-link", :lld_link, "tools"))
-        push!(products, ExecutableProduct("wasm-ld", :wasm_ld, "tools"))
+        push!(products, ExecutableProduct("lld", :lld, "bin"))
```

Comment:
We need to keep these here, we just need to figure out where they are for any given LLVM version.

Reply:
![image](https://user-images.githubusercontent.com/17990405/181905539-350274e6-d5b1-4801-bbc2-784689701199.png)

They are present in `tools/` dir, but the symlinks are invalid. We fix them in rocBLAS, for example, but manually symlinking `ln -s ${prefix}/bin/lld ${prefix}/tools/lld`.

Reply:
Yeah, then let's fix them in this build script.

Reply:
Fixed in the latest commit.